mypyc/genops.py

```diff
@@ -46,7 +46,7 @@ def f(x: int) -> int:
     ENV_ATTR_NAME, NEXT_LABEL_ATTR_NAME, TEMP_ATTR_NAME, LAMBDA_NAME,
     MAX_SHORT_INT, TOP_LEVEL_NAME
 )
-from mypyc.freevariables import FreeVariablesVisitor
+from mypyc.prebuildvisitor import PreBuildVisitor
```

Comment:
I decided to rename this since this traverse does a lot more than just determine free variables now. More so, it kind of does a pre-build analysis. Let me know if you think there's a better name, but I think we can definitely do better than `FreeVariablesVisitor`.